src/org/zaproxy/zap/extension/pscanrulesBeta/CookieLooselyScopedScanner.java

```diff
@@ -136,7 +136,7 @@ private void raiseAlert(HttpMessage msg, int id, String host, List<HttpCookie> l
 		}		
 		
 		alert.setDetail(getDescriptionMessage(), msg.getRequestHeader()
-				.getURI().toString(), null, getExploitMessage(), 
+				.getURI().toString(), null, "", 

```

Comment:
Even if it was not simple, unused code should be removed (specially if it's outdated/non-functional).